cmd/basecoin/commands/init.go

```diff
@@ -64,7 +67,24 @@ func initCmd(cmd *cobra.Command, args []string) error {
 		return errors.New("Address must be 20-bytes in hex")
 	}
 
-	genesis := GetGenesisJSON(viper.GetString(FlagChainID), userAddr)
+	var options []string
+	var optionsStr string
+	sep := ",\n      "
+	optionsRaw := viper.GetStringSlice(FlagOption)
+	if len(optionsRaw) > 0 {
```

Comment:
How about pulling out the logic into its own function so that we can write a unit test against it. This applies to L70 - L85.

Reply:
Originally, I had thought to unit test the way which you have described, but I kind of figured it should be tested like the rest of our commands as a part of the bash testing... (which has been done checkout init-server.sh). Additionally, the results are more relevant in the context of the genesis file as opposed to just a lone string to add in... Anyways, we can do it, not sure if it makes sense since we have bash test

Reply:
I agree with Rigel, this cli parsing stuff is covered by the bash tests.  Ugly but there is not really a better way.

Reply:
okay leaving